lib/ansible/modules/uri.py

```diff
@@ -392,6 +392,13 @@
 
 JSON_CANDIDATES = ('text', 'json', 'javascript')
 
+# List of response key names we do not want sanitize_keys() to change.
+NO_MODIFY_KEYS = frozenset(
+    ('msg', 'exception', 'warnings', 'deprecations', 'failed', 'skipped',
+     'changed', 'rc', 'stdout', 'stderr', 'elapsed', 'path', 'location',
+     'content_type')
```

Comment:
We aren't likely to have some of these, like `rc`, `stdout`, but it's not harmful I don't think, and isn't a required change.  In fact it's probably nice, as it gives others a good list to look at.

Reply:
Right, it's not harmful, and is instructive.

Reply:
Would we only want these keys to be skipped at the toplevel?

Reply:
I don't think so. A user may want to specify keys that might appear at any depth, and maybe even multiple times. And it wouldn't make sense to hide a no_log value in a key at the top level, but not at a lower level, since that defeats the purpose.

Reply:
Err, I meant "hide at a lower level but not at the top level", but I hope you see what I meant.

Reply:
<nod>  Roger that.